composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "prooph/psb-zeromq-producer",
+    "description": "ZeroMQ Message Producer for Prooph Service Bus",
+    "homepage": "https://github.com/bweston92/psb-zeromq-producer",
+    "license": "BSD-3-Clause",
+    "authors": [
+        {
+            "name": "Bradley Weston",
```

Comment:
Please add @codeliner as author (even if he's not yet). He will provide support in future.